courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>add PROJECT variable</p><p>os.environ["PROJECT"] = PROJECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>PROJECT is not being used anywhere in bash below, did it not run without it for you? I tested specifically for this earlier and had no issues.</p>

Reply:
ah yeah. Just added it out of habit, but yeah not needed later.